Playbooks/playbook-Phishing_Investigation_-_Generic_4.0.yml

```diff
@@ -0,0 +1,859 @@
+id: Phishing Investigation - Generic
+version: -1
+name: Phishing Investigation - Generic
+fromversion: 4.0
```

Comment:
4.0.0 thats what causing your build to fail. please check in other locations as well.